extensions/s3/src/test/java/io/deephaven/extensions/s3/S3SeekableChannelMinIOTest.java

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (c) 2016-2023 Deephaven Data Labs and Patent Pending
+ */
+package io.deephaven.extensions.s3;
+
+
+import io.deephaven.extensions.s3.S3Instructions.Builder;
+import io.deephaven.extensions.s3.SingletonContainers.MinIO;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Tag;
+import software.amazon.awssdk.services.s3.S3Client;
+
+@Tag("testcontainers")
+public class S3SeekableChannelMinIOTest extends S3SeekableChannelTestBase {
```

Comment:
I haven't been able to dig into this further yet. It may be a specific issue w/ how Docker Desktop on Mac exposes connectivity, or something else.

Given that this is meant to be a testOutOfBand / nightly thing, do you see this as a blocker to merging? It shouldn't effect any developers unless they are explicitly wanting to do local integration testing on Mac currently.

I can file an issue to follow up?

Reply:
Can you do an OS-specific short-circuit in the test for now, and file a follow-up issue?